lhotse/recipes/spatial_librispeech.py

```diff
@@ -0,0 +1,260 @@
+import logging
+from pathlib import Path
+from typing import Dict, Optional, Sequence, Union
+
+from tqdm.auto import tqdm
+
+from lhotse import fix_manifests, validate_recordings_and_supervisions
+from lhotse.audio import RecordingSet
+from lhotse.supervision import SupervisionSegment, SupervisionSet
+from lhotse.utils import Pathlike, resumable_download
+
+try:
```

Comment:
Please move those try/except to local imports instead (inside download/prepare functions). Otherwise anybody importing lhotse without these dependencies will get these exceptions.